taxonomies/additives.txt

```diff
@@ -7819,68 +7991,121 @@ nl:E333(ii), Dicalciumcitraat
 pl:E333(ii), Cytrynian diwapniowy
 pt:E333(ii), Citrato dicálcico
 ro:E333(ii), Citrat dicalcic
+sh:E333(ii), Dikalcijum citrat
 sk:E333(ii), Citran divápenatý (dicitran divápenatý), Kalcium-citrát
 sl:E333(ii), Dikalcijev citrat
+sr:Dikalcijum citrat
 sv:E333(ii), Dikalciumcitrat
+# azb:E333(ii), دیکلسیوم سیترات
 e_number:en:333
 wikidata:en:Q420280
 additives_classes:en: en:sequestrant, en:stabilizer
 organic_eu:en:authorized
 
 <en:E333
 en:E333(iii), Tricalcium citrate
+ar:E333(iii), سترات الكالسيوم
 bg:E333(iii), Трикалциев цитрат
-cs:E333(iii), Tricalcium citrate
+cs:E333(iii), Tricalcium citrate, Citrát vápenatý
 da:E333(iii), Tricalciumcitrat
 de:E333(iii), Tricalciumcitrat, Calciumcitrat
 el:E333(iii), Κιτρικο ασβεστιο
-es:E333(iii), Citrato tricálcico
+es:E333(iii), Citrato tricálcico, Citrato de calcio
 et:E333(iii), Kaltsiumtsitraat
+fa:E333(iii), کلسیم سیترات
 fi:E333(iii), Trikalsiumsitraatti
-fr:E333(iii), Citrate tricalcique
-hu:E333(iii), Trikalcium-citrát
-it:E333(iii), Citrato tricalcico
+fr:E333(iii), Citrate tricalcique, Citrate de calcium
```

Comment:
why add Citrate de calcium here? It is already on E333, it should not be on E333(iii)

Reply:
The E333 is the generic name, which is in plural, and can be either of the three others. I.e. a Calcium atom with a citrate molecule.

The E333iii is the version without any hydrogen, so that three Calcium atoms can be bound. That is why it is called Tricalcium. It seems that people find the tri to long and shorten it to Calciumcitrate (note the singular).

By the way this also true for some other E-numbers.

Reply:
Ok.